content/guides/getstarted/how-to-create-a-token.md

```diff
@@ -0,0 +1,329 @@
+```
+
+You can also add additional metadata to your token as a key-value pair like
+this:
+
+```bash
+spl-token update-metadata niceG6oxHmPcXVdNaUoECzWXn8Jz8fA5Q99QauJ4Gun niceness 100%
+```
+
+This is more useful for NFTs but you can also use it for fungible tokens.
+
+## Mint tokens
+
+Now that the token is completely set up we can mint some tokens. Let's mint 100.
+
+First we need to create a token account to hold the tokens for our token mint:
+
+```bash
+spl-token create-account niceG6oxHmPcXVdNaUoECzWXn8Jz8fA5Q99QauJ4Gun
+```
+
+This will create a new token account for the account that is currently set in
+the solana config. You can also specify a different account by adding the
+address at the end of the command.
+
+And now we can finally mint some tokens into that token account:
+
+```bash
+spl-token mint niceG6oxHmPcXVdNaUoECzWXn8Jz8fA5Q99QauJ4Gun 100
+```
+
+Now you can also send the token to your friends, for example:
+
+```bash
+spl-token transfer niceG6oxHmPcXVdNaUoECzWXn8Jz8fA5Q99QauJ4Gun 10 \
+friendAddress --fund-recipient
+```
+
+"--fund-recipient" means that you will pay to create the token account for your
+friend.
+
+## Creating token markets
+
+Now that your token is created you probably want to create a market and a pool
+for it and have it listed on exchanges. There are many ways to create a market
+for your token. Here are some possible solutions:
+
+- [Radium](https://docs.raydium.io/raydium/pool-creation/creating-a-standard-amm-pool)
+- [Orca](https://docs.orca.so/orca-for-liquidity-providers/classic-community-listing/creating-a-pool-tutorial)
+- [Meteora](https://docs.meteora.ag/liquidity-primitives/dynamic-amm-pools/permissionless-dynamic-pools)
+- [Mango Markets](https://docs.mango.markets/)
+- [Jupiter](https://station.jup.ag/docs/get-your-token-onto-jup)
+
+If you have your own token market platform feel free to open a PR to add it here
+using the edit page button.
+
+## Community tools to create tokens
+
+Use these at your own risk. These tools make it easy to create tokens, but may
+also expose you to security risks, add additional fees, and could potentially
+give the tool creators control over your token.
+
+- [Smithii](https://smithii.io/)
+- [Smithii Tutorial](https://smithii.io/en/create-meme-coin-solana/)
+- [Fluxbeam](https://fluxbeam.xyz/)
+- [Fluxbeam Tutorial](https://medium.com/@Fluxbeam/how-to-create-a-token-using-fluxbeam-and-solana-token-extensions-9d1aaa8d98ea)
+
+If you have your own token launch platform feel free to open a PR to add it
+using the edit page button.
+
+## Further reads about tokens
+
+- [Token extensions](https://solana.com/developers/guides/token-extensions/getting-started)
+- [External: a16z Rules on Token Launches](https://a16zcrypto.com/posts/article/5-rules-for-token-launches/)
+- [External: a16z Risks](https://a16zcrypto.com/posts/article/navigating-token-launch-risks/)
+- [External: a16z Getting Ready](https://a16zcrypto.com/posts/article/getting-ready-to-launch-a-token/)
```

Comment:
These a16z links are okay, but lets add some context around them and why they might be helpful

Reply:
I added some context. But after what we talked yesterday im not sure anymore if we should keep them in. Wdyt